src/font.ml

```diff
@@ -1,3 +1,5 @@
+open Stdlib
```

Comment:
Why is this file changed at all? This has nothing to do with the task you're working on? You should not change code you're not working on, as it will cause difficulty for people who are working on it.